Dockerfile.local

```diff
@@ -1,79 +1,87 @@
-#RUN git clone https://github.com/threagile/threagile.git
-COPY . /app/threagile
 
+# Cache build dependencies
+COPY go.mod go.sum ./
+RUN go mod download
 
+COPY . .
 
+# Set build-time variables
+ARG GOOS=linux
 
-######
-## Stage 2: Build application with Go's build tools
-######
-FROM golang as build
-ENV GO111MODULE=on
-# https://stackoverflow.com/questions/36279253/go-compiled-binary-wont-run-in-an-alpine-docker-container-on-ubuntu-host
-#ENV CGO_ENABLED=0 # cannot be set as otherwise plugins don't run
-WORKDIR /app
-COPY --from=clone /app/threagile /app
-RUN go version
-RUN go test ./...
-RUN GOOS=linux go build -a -trimpath -ldflags="-s -w -X main.buildTimestamp=$(date '+%Y%m%d%H%M%S')" -buildmode=plugin -o raa.so raa/raa/raa.go
-RUN GOOS=linux go build -a -trimpath -ldflags="-s -w -X main.buildTimestamp=$(date '+%Y%m%d%H%M%S')" -buildmode=plugin -o dummy.so raa/dummy/dummy.go
-RUN GOOS=linux go build -a -trimpath -ldflags="-s -w -X main.buildTimestamp=$(date '+%Y%m%d%H%M%S')" -buildmode=plugin -o demo-rule.so risks/custom/demo/demo-rule.go
-RUN GOOS=linux go build -a -trimpath -ldflags="-s -w -X main.buildTimestamp=$(date '+%Y%m%d%H%M%S')" -o threagile
-# add the -race parameter to go build call in order to instrument with race condition detector: https://blog.golang.org/race-detector
+RUN : \
+    && go version \
+    && go test ./...
 
+# Build plugins and threagile
+RUN : \
+    && go build -a -trimpath -ldflags="-s -w -X main.buildTimestamp=$(date '+%Y%m%d%H%M%S')" -buildmode=plugin -o raa.so raa/raa/raa.go \
+    && go build -a -trimpath -ldflags="-s -w -X main.buildTimestamp=$(date '+%Y%m%d%H%M%S')" -buildmode=plugin -o dummy.so raa/dummy/dummy.go \
+    && go build -a -trimpath -ldflags="-s -w -X main.buildTimestamp=$(date '+%Y%m%d%H%M%S')" -buildmode=plugin -o demo-rule.so risks/custom/demo/demo-rule.go \
+    && go build -a -trimpath -ldflags="-s -w -X main.buildTimestamp=$(date '+%Y%m%d%H%M%S')" -o threagile
 
+# Copy files in build image to simplify copy instruction
+RUN : \
+    && cp /app/demo/example/threagile.yaml /app/demo/example/threagile-example-model.yaml \
+    && cp /app/demo/stub/threagile.yaml /app/demo/stub/threagile-stub-model.yaml
 
 
 ######
-## Stage 3: Make final small image
+## Stage 2: Make final small image
 ######
-FROM alpine
+FROM docker.io/library/alpine:latest
 
-# label used in other scripts to filter
+# Label used in other scripts to filter
 LABEL type="threagile"
 
-# add certificates
-RUN apk add ca-certificates
-# add graphviz, fonts
-RUN apk add --update --no-cache graphviz ttf-freefont
-# https://stackoverflow.com/questions/66963068/docker-alpine-executable-binary-not-found-even-if-in-path
-RUN apk add libc6-compat
-# https://stackoverflow.com/questions/34729748/installed-go-binary-not-found-in-path-on-alpine-linux-docker
-# RUN mkdir -p /lib64 && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
-# clean apk cache
-RUN rm -rf /var/cache/apk/*
+RUN apk add --no-cache \
```

Comment:
It would be nice to know the context why adding libc6-compat (```RUN apk add libc6-compat```) was removed. As far as I can see that was a workaround, is adding everything withing one ```RUN apk add``` fixing the problem and how?

Reply:
I changed the building step to run in `golang:alpine` instead of the default `golang` (which is based on Debian). Because of this we do not need `libc6-compat` as a compatibility layer for glibc anymore.